com.github.rssguard.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "com.github.rssguard",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "rssguard",
+    "rename-icon": "rssguard",
+    "finish-args": [ "--share=ipc",
+                     "--share=network",
+                     "--socket=x11",
+                     "--socket=wayland",
```

Comment:
My understanding is that the current KDE recommendation is not to use Wayland support with Flatpak, but to fall back to XWayland for a better experience. This may have changed so it's just a suggestion.

Reply:
Using "--socket=wayland" only allows the app to work with wayland, right? It doesn't say anything about whether the app uses Wayland or XWayland. Also, we handle this in flatpak platform plugin, where we decide to use Wayland or XCB (for X11) platform plugin. We actually recently added support for Wayland into flatpak platform plugin and also additional fixes to make Wayland work under flatpak (with flatpak platform plugin).

Reply:
Right, although not allowing an app to see wayland normally makes it use an X fallback path. But really excited to hear that this is now being solved in the platform plugin, very happy to see that.